flight/UAVObjects/uavobjectmanager.c

```diff
@@ -1651,6 +1676,19 @@ static int32_t sendEvent(struct UAVOBase * obj, uint16_t instId,
 	LL_FOREACH(obj->next_event, event) {
 		if (event->eventMask == 0
 			|| (event->eventMask & triggered_event) != 0) {
+			if (event->hasThrottle) {
```

Comment:
Might be nice to have a comment here describing what we expect from the if loop.